lib/CL/pocl_debug.h

```diff
@@ -497,6 +497,16 @@ POCL_EXPORT
 
 #endif
 
+#define POCL_RETURN_ERROR(err_code, ...)                                    \
+    POCL_MSG_ERR2(#err_code, __VA_ARGS__); return err_code; \
```

Comment:
```suggestion
#define POCL_RETURN_ERROR(err_code, ...)                                    \
    POCL_MSG_ERR2(#err_code, __VA_ARGS__); return err_code
```
I see there are `POCL_RETURN_ERROR(...);` calls. MSVC C compiler treats `;;`s as error.